lib/matplotlib/axes/_base.py

```diff
@@ -2540,27 +2540,21 @@ def in_axes(self, mouseevent):
         return self.patch.contains(mouseevent)[0]
 
     def get_autoscale_on(self):
-        """
-        Get whether autoscaling is applied for both axes on plot commands
-        """
+        """Return True if all axes are autoscaled, False otherwise."""
```

Comment:
`axes` was, and is, confusing here. Not sure how best to write it.

Reply:
Ah, missed that.  Replaced by "each axis".

Reply:
Yes, that works.